web-ui/src/main/resources/catalog/locales/ca-core.json

```diff
@@ -446,7 +446,7 @@
     "listTypeSmall": "Switch to Small list",
     "layersWithWhiteSpaces": "Some layers on this map contains white spaces on their name. Layers with white spaces may not be properly printed.",
     "unselected": "unselected",
-    "indexingTaskUndefined": "Unknow indexing task (abnormal state)",
+    "indexingTaskUndefined": "Unknown indexing task (abnormal state)",
```

Comment:
Not needed. Only English language has to be updated in github. Translations are all coming from transifex only.

Reply:
I removed these fix for non english